addons/popochiu/engine/objects/prop/popochiu_prop.gd

```diff
@@ -137,6 +137,65 @@ func get_total_frames() -> int:
 	return frames * v_frames
 
 
+#endregion
+
+#region AnimationPlayer ############################################################################
+## Will play the Popochiu Prop Animation Player [param name] animation on the Popochiu 
+## Prop if the [prop name] animation exists.
+## Optionally you can use other AnimationPlayer options, see Godot AnimationPlayer documentation for
+## more details.
+func play_animation(name: StringName = &"", custom_blend: float = -1, custom_speed: float = 1.0, from_end: bool = false) -> void:
+	if not has_node("AnimationPlayer"): return
+	$AnimationPlayer.play(name, custom_blend, custom_speed, from_end)
+
+## Will Play the Popochiu Prop Animation Player [param name] animation backwards on the Popochiu
+## Prop if the [prop name] animation exits.
+## This method is a shorthand for play_animation() with custom_speed = -1.0 and from_end = true
+func play_animation_backwards(name: StringName = &"", custom_blend: float = -1) -> void:
+	if not has_node("AnimationPlayer"): return
+	$AnimationPlayer.play_backwards(name, custom_blend)
+
+
+## Will stop the animation that is currently playing on the Popochiu Prop.
+## The animation position is reset to 0 and the custom_speed is reset to 1.0
```

Comment:
```suggestion
## Will stop the animation that is currently playing on the Popochiu Prop.
## The animation position is reset to 0 and the custom_speed is reset to 1.0.
## Set [param keep_state] to true to avoid the animation to be updated visually.
```